src/containers/audio-rewards-page/components/ClaimRewardButton.tsx

```diff
@@ -5,29 +5,29 @@ import { useDispatch, useSelector } from 'react-redux'
 
 import { useModalState } from 'common/hooks/useModalState'
 import { getAccountUser, getUserHandle } from 'common/store/account/selectors'
+import { getHCaptchaStatus } from 'common/store/pages/audio-rewards/selectors'
 import { useRemoteVar } from 'containers/remote-config/hooks'
 import { useScript } from 'hooks/useScript'
 import AudiusBackend from 'services/AudiusBackend'
 import { IntKeys, StringKeys } from 'services/remote-config'
 import { COGNITO_SCRIPT_URL } from 'utils/constants'
 import { encodeHashId } from 'utils/route/hashIds'
 
-import {
-  ClaimStatus,
-  CognitoFlowStatus,
-  getHCaptchaStatus,
-  HCaptchaStatus,
-  setClaimStatus,
-  setCognitoFlowStatus
-} from '../store/slice'
 import {
   ChallengeRewardID,
   FailureReason,
   FlowErrorEvent,
   FlowSessionEvent,
   FlowUICloseEvent,
   FlowUIOpenEvent
-} from '../types'
+} from '../../../common/models/AudioRewards'
```

Comment:
Might be nicer to use absolute paths here
```suggestion
} from 'common/models/AudioRewards'
```